dashboard/src/shared/types.ts

```diff
@@ -387,6 +387,8 @@ export interface IBasicFormParam {
   sliderMin?: number;
   sliderMax?: number;
   sliderUnit?: string;
+  disable?: string;
+  enable?: string;
```

Comment:
Similar to my comment in the last PR, not sure if `IBasicFormEnableDisableParam` composing `IBasicFormParam` would mean we don't have to mix in slider-specific and enable/disable specific fields here (and in the future).

Reply:
as I said, I am not sure that would really improve the code but let me send a follow up PR and we can discuss that there.